src/test/fuzz/fuzz.cpp

```diff
@@ -56,6 +57,23 @@ static bool read_stdin(std::vector<uint8_t>& data)
 }
 #endif
 
+#if defined(PROVIDE_FUZZ_MAIN_FUNCTION)
+static bool read_file(std::filesystem::path p, std::vector<uint8_t>& data)
```

Comment:
```suggestion
static bool read_file(fs::path p, std::vector<uint8_t>& data)
```

The wrapper should work fine here?

Reply:
Yup, that worked, thanks!